Button in the header to expand all items

tophf opened the pull request:
Now it's possible to read the frequently updated feeds much faster - right from the popup.

olsh commented:
Thank you for the contribution.
Please make it as a toggle button.

tophf commented:
Done.

tophf commented:
| Feature | Note |
|--------|---------|
| 1. Button in the header to expand all items | |
| 2. Condensed display mode | Only show titles and icons, expand the blog name when mouse is over the icon |
| 3. Click-an-item-anywhere | Clicking a title/content expands or collapses the item. When collapsing, it's scrolled into view if needed. |

tophf commented:
Two more commits that fix two big issues in Feedly Notifier:
1. content links opening with expand-all-on-show option
2. Show popup immediately, fetch/expand items afterwards - eliminates unresponsiveness of the extension button in case of network timeouts or lots of feeds, usually it takes more than a second or even a few seconds when the autoexpand option is enabled

olsh commented:
Good job, thank you.
But there is still the bug with opening blog link.
When you trying to open a blog link, a new blank tab is opened.

tophf commented:
Fixed the blog link regression.
Added custom CSS, now it's possible to finetune appearance:
```css
.article-title a {text-decoration: none}
.expanded .article-title a {font-size:1.3em}
```

tophf commented:
Needs some polishing, apparently. When "show-expanded-by-default" options is enabled it's not always smooth. I'll post a fixup once I pinpoint the cause.

tophf commented:
Looks good.
